### PR TITLE
Add how_to on the keys selection

### PR DESCRIPTION
**Pull request by hurricane642**
Hi all, this PR contains a how_to example for the new attribute of calculator `selected_keys`, added in #134

### CODE REVIEW

File: docs/src/how-to/keys-selection.rst [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+            .. container:: sphx-glr-download sphx-glr-download-python
+
+                :download:`Download Python source code for this example: keys-selection.py <../examples/keys-selection.py>`
+
+            .. container:: sphx-glr-download sphx-glr-download-jupyter
+
+                :download:`Download Jupyter notebook for this example: keys-selection.ipynb <../examples/keys-selection.ipynb>`
+
+        .. include:: ../examples/keys-selection.rst
```
Comment (Luthaf) [inline docs/src/how-to/keys-selection.rst:34]:
```suggestion
        .. include:: ../examples/keys-selection.py
```

File: python/examples/keys-selection.py [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+calculator = SoapPowerSpectrum(**HYPER_PARAMETERS)
+
+descriptor = calculator.compute(frames)
+
+# %%
+#
+# The selections for keys can be a set of ``Labels``, in which case the names
+# of the keys must be a set of the names of the keys produced by the
+# calculator. You can see the default set of names with:
```
Comment (Luthaf) [inline python/examples/keys-selection.py:47]:
```suggestion
# The selections for keys can be a set of ``Labels``, with the names
# of the keys being a subset of the names of the keys produced by the
# calculator.
```

File: python/examples/keys-selection.py [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+
+print("properties names:", descriptor.keys.names)
```
Comment (Luthaf) [inline python/examples/keys-selection.py:49]:
```suggestion
descriptor = calculator.compute(frames)
print("keys names:", descriptor.keys.names)
```

File: python/examples/keys-selection.py [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+        "Gto": {},
+    },
+    "cutoff_function": {
+        "ShiftedCosine": {"width": 0.5},
+    },
+}
+
+calculator = SoapPowerSpectrum(**HYPER_PARAMETERS)
+
+descriptor = calculator.compute(frames)
```
Comment (Luthaf) [inline python/examples/keys-selection.py:41]:
```suggestion
```

**Review by Luthaf — COMMENTED** [review]
Looks good overall!

**Comment by Luthaf** [comment]
I did some minor changes and phrasing improvements, let me know if that works for you!

**Comment by hurricane642** [comment]
> I did some minor changes and phrasing improvements, let me know if that works for you!

Yes, it looks great!

**Pull request merged**